### PR TITLE
terminal: fill all available space + handle resizes

### PR DESCRIPTION
**Pull request by bjia56**

### CODE REVIEW

**Comment by bjia56** [comment]
The div that holds the actual xterm isn't resized properly

File: src/components/PtyComponent.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-card>
+  <v-card :style="fillScreen ? { height: '100%', display: 'flex', flexDirection: 'column' } : {}">
```
Comment (koush) [inline src/components/PtyComponent.vue:2]: don't do this, just apply this change on the terminal component itself, which is the only thing that styles it.

File: src/components/Terminal.vue [hunk 2]
```diff
@@ -1,10 +1,6 @@
 <template>
   <v-container fluid>
-    <v-row >
-      <v-col cols="12">
-        <PtyComponent title= "Terminal" native-id="terminalservice" control></PtyComponent>
-      </v-col>
-    </v-row>
+    <PtyComponent title= "Terminal" native-id="terminalservice" control fill-screen></PtyComponent>
```
Comment (koush) [inline src/components/Terminal.vue:3]: apply the style here.

File: src/components/PtyComponent.vue (hunk 1 above)
Comment (koush) [reply src/components/PtyComponent.vue:2]: generally I leave the top level component alone when it comes to layout and leave that to the parent container. styling the top level component can cause issues with reuse.

**Comment by bjia56** [comment]
I was able to move the conditional styling out but the xtermjs box still needs a height=100% for it to flex properly. From what I see, it doesn't affect the existing console/repl windows

**Pull request merged**